manual/manual/cmds/intf-c.etex

```diff
@@ -182,26 +182,40 @@ section~\ref{s:c-custom}).}
 \entree{"caml/intext.h"}{operations for writing user-defined
 serialization and deserialization functions for custom blocks
 (see section~\ref{s:c-custom}).}
+\entree{"caml/signals.h"}{runtime lock, signals, and other
+  asynchronous callbacks (see
+  section~\ref{ss:c-process-pending-actions}).}
```

Comment:
@gadmm , would you mind If I pulled this documentation update in a separate PR?

Reply:
No problem, take all you need and add it to your PR